client/src/components/ActivityList/ActivityList.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import ActivityList from "./ActivityList";
+
+let momActivityList = ["Spa", "Wine Nights", "Brunch with Other Moms"];
+let kidsActivityList = [
+  "Snorkeling",
+  "Hiking",
+  "Island Hopping",
+  "Hawaiian Luaus",
+  "Food Markets",
+  "Beaches"
+];
+
+storiesOf("ActivityList", module)
+  .add("Activities for Moms", () => (
+    <ActivityList type="mom" label="Activities for Moms" momActivityList={momActivityList} />
```

Comment:
Can you get rid of the `type` property? There isn't really a `type` of activity list - just an activity list with different text for the header and a different color

Reply:
That said, can you add a `color` property that controls the color?